package.json

```diff
@@ -56,13 +56,15 @@
   },
   "dependencies": {
     "bluebird": "^3.4.7",
```

Comment:
Right now the code depends on http://bluebirdjs.com/docs/api/promise.promisify.html